src/io/flutter/util/ModuleUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.util;
+
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.roots.ModuleRootManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import io.flutter.module.FlutterModuleType;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Collection;
+
+public class ModuleUtil {
+
+  public static  @Nullable VirtualFile getProjectContentRoot(@NotNull Module module) {
```

Comment:
IntelliJ's formatter places annotations on a separate line, so we prefer to write it before `public static`, otherwise formatting looks not ideal.